<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcpcilent</class>
 <widget class="QDialog" name="tcpcilent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>500</width>
     <height>244</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter2">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>主机</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit"/>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>接收内容</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser"/>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>129</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="text">
     <string>TCP客户端</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>501</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
